frontend/src/components/PrivateRoute/PrivateRoute.js

```diff
@@ -1,13 +1,15 @@
 import { Redirect, Route } from "react-router-dom";
+import { ERROR_MESSAGE } from "../../constants/messages";
+import PATH from "../../constants/path";
 import useTokenContext from "../../hooks/useTokenContext";
 
 const PrivateRoute = ({ children, ...props }) => {
   const { token } = useTokenContext();
 
-  if (!token) alert("로그인이 필요합니다.");
+  if (!token) alert(ERROR_MESSAGE.ACCESS.REQUIRED_LOGIN);
 
   return (
-    <Route {...props}>{token ? children : <Redirect to="/login" />}</Route>
+    <Route {...props}>{token ? children : <Redirect to={PATH.LOGIN} />}</Route>
```

Comment:
```suggestion
    token ? <Route {...props} /> : <Redirect to={PATH.LOGIN} />
```
Route의 자식에 Redirect가 있는게 조금 어색한 것 같아요
children 도 rest parameter에 포함시켜서 위처럼 바꿔주면 좀 더 가독성이 좋아질 것 같아서 제안해봅니다.

Reply:
생각못한 부분이네요! 좋습니다.